.travis.yml

```diff
@@ -4,13 +4,12 @@ sudo: false
 
 matrix:
   include:
-    - php: 5.2
-    - php: 5.3
```

Comment:
big :-1: for that

1. this project officially supports 5.2+, dropping that in CI will lead to introduce changes, which are not 5.2/5.3 compatible.
2. even if it would be fine to drop 5.2/5.3 support, take a look, you are removing them from matrix, but keep conditions for 5.2 in script/install